apps/scan/frontend/src/components/misvote_warnings/test_utils.test.ts

```diff
@@ -0,0 +1,23 @@
+import { electionSample } from '@votingworks/fixtures';
+import { AnyContest } from '@votingworks/types';
+
+const CONTEST_TEMPLATE: AnyContest = electionSample.contests[0];
+
+export function generateContests(count: number): AnyContest[] {
+  const contests: AnyContest[] = [];
+
+  for (let i = 0; i < count; i += 1) {
+    contests.push({
+      ...CONTEST_TEMPLATE,
+      id: `${i}`,
+      title: `[TEST ${i}] ${CONTEST_TEMPLATE.title}`,
+    });
+  }
+
+  return contests;
+}
+
+test('generates contests', () => {
+  const contests = generateContests(2);
+  expect(contests[0]).not.toEqual(contests[1]);
```

Comment:
I think `toMatchObject` would probably be more complete here, to ensure they're not different objects with the same contents.

Reply:
ah, IIUC `toMatchObject()` only matches object subsets, so `{a: "one", b: "two"}` would match `{a: "one"}`, whereas `toEqual()` matches all fields (and `toBe()` matches the actual object references)